.github/workflows/dokken-unit-tests.yml

```diff
@@ -3,60 +3,6 @@ name: ParallelCluster Cookbook Unit Tests on Docker
 on: [workflow_dispatch]
```

Comment:
I wonder if we still need this workflow at all. We never used it and often forgot to maintain.

Reply:
yeah I think we can remove it but I'll do it in a separate patch.